pkg/minikube/cluster/cluster.go

```diff
@@ -371,16 +373,34 @@ func GetHostDockerEnv(api libmachine.API) (map[string]string, error) {
 }
 
 // GetHostLogs gets the openshift logs of the host VM.
-func GetHostLogs(api libmachine.API) (string, error) {
+// If follow is specified, it will tail the logs
+func GetHostLogs(api libmachine.API, follow bool) (string, error) {
 	host, err := CheckIfApiExistsAndLoad(api)
 	if err != nil {
 		return "", err
 	}
-	s, err := host.RunSSHCommand(logsCmd)
-	if err != nil {
+
+	cmdToRun := logsCmd
+	if follow {
+		cmdToRun = logsCmdFollow
+		c, err := host.CreateSSHClient()
+		if err != nil {
+			return "", err
+		}
+
+		err = c.Shell(cmdToRun)
+		if err != nil {
+			return "", errors.Wrap(err, "Error creating ssh client")
+		}
+
 		return "", nil
 	}
-	return s, err
+
+	s, err := host.RunSSHCommand(cmdToRun)
+	if err != nil {
+		return "", err
+	}
+	return s, nil
```

Comment:
Couple of suggestions. 

* I think `host.RunSSHCommand(cmdToRun)` should be in an else statement as it is mutually exclusive to `if follow` code block. I do not see  `s, err := host.RunSSHCommand(cmdToRun)` is getting executed if user has specified -f. 
* Also we can write the code without `cmdToRun` variable.